sdk/search/Azure.Search.Documents/apis.txt

```diff
@@ -165,11 +162,10 @@ namespace Azure.Search.Documents.Models
     public class IndexDocumentsAction { }
     public class IndexDocumentsBatch<T> { }
     public class IndexDocumentsBatch { }
+}
 
-
-
-
-
+namespace Azure.Search.Documents.Indexes.Models
```

Comment:
If we don't put stuff in .Indexes like the client I mentioned above, is it worth having .Indexes.Models? I was imaging - if we keep all clients in the top-level - indexing models would just go in .Indexes sibling to .Models. Just a thought.

Reply:
All the clients except `SearchClient` and `SearchServiceClient` are in `.Indexes`.

Reply:
There are two swaggers, index and service. Can we just simply have different swagger in different namespace for auto-gen layer. The top clients follow what you design above.

Reply:
That would probably be a very effective way of making this change.